docs/runbooks/LowVirtControllersCount.md

```diff
@@ -16,22 +16,22 @@ In addition, if another `virt-launcher` instance terminates unexpectedly, KubeVi
 
 ## Diagnosis
 1. Set the NAMESPACE environment variable:
-    ```
-    $ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
-    ```
+```bash
+$ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+```
 
 2. Verify that running `virt-controller` pods are available:
-    ```
-    $ kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-controller
-    ```
+```bash
+$ kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-controller
+```
 
 3. Check whether any `virt-controller` pods have terminated unexpectedly or are in a `NotReady` state:
```

Comment:
I think this should be:
3. Check the `virt-launcher` logs for error messages:
4. Obtain the details of the `virt-launcher` pod to check for status conditions such as unexpected termination or a `NotReady` state.